jdi-light-mobile-bootstrap-web-tests/README.md

```diff
@@ -0,0 +1,110 @@
+#How to launch HTML Web tests on Android emulator or IOS simulator
+## Requirements for Android
+
+Required tools: Android Studio, Appium, Maven.<br>
+1. Download **chrome driver** For more details
```

Comment:
Strange numebring all points are 1. Please fix this

Reply:
In markdown all numbered items have number 1.